### PR TITLE
fix #6761 move reportId to plot internals

### LINKED ISSUE #6761: move reportId input plotting API
The reportId parameter is passed into reportPanel, reportHeading, reportContent, but is really used internally to the plots to manage the report overlay. I think the plots can manage this state, rather than maintaining it outside the panels.

### PR DESCRIPTION
**Pull request by moellep**

### CODE REVIEW

File: sirepo/package_data/static/js/sirepo-plotting.js [hunk 1]
```diff
@@ -970,6 +970,8 @@ SIREPO.app.factory('plotting', function(appState, frameCache, panelState, utilit
             };
         },
 
+        randomId: () => Math.floor(Math.random() * Number.MAX_SAFE_INTEGER),
+
```
Comment (mkeilman) [inline sirepo/package_data/static/js/sirepo-plotting.js:974]: Might as well use `SIREPO.UTILS.randomString()`

**Review by mkeilman — COMMENTED** [review]
One suggestion around the random id, otherwise lgtm

File: sirepo/package_data/static/js/sirepo-plotting.js (hunk 1 above)
Comment (moellep) [reply sirepo/package_data/static/js/sirepo-plotting.js]: I added SIREPO.UTILS.randomId() which uses randomString(). The HTML id spec recommends starting ids with a letter.

Comment (mkeilman) [reply sirepo/package_data/static/js/sirepo-plotting.js]: Cool

**Review by mkeilman — APPROVED** [review]

**Pull request merged**